velox/exec/AggregationMasks.cpp

```diff
@@ -37,6 +37,7 @@ void AggregationMasks::addInput(
 
     // Get the projection column vector that would be our mask.
     const auto& maskVector = input->childAt(entry.first);
+    VELOX_CHECK_EQ(maskVector->type(), BOOLEAN(), "Mask must be BOOLEAN");
```

Comment:
Wondering if we can make the failure message more descriptive so that query owner can troubleshoot the query.

Reply:
Yeah maybe mention `FILTER(WHERE ...)` clause